MonoGame.Framework/Graphics/SpriteBatch.cs

```diff
@@ -394,6 +408,10 @@ void CheckValid(SpriteFont spriteFont, StringBuilder text)
 					_texCoordTL, 
 					_texCoordBR);			
 			
+		    unchecked
+		    {
+                ++_drawCount;
```

Comment:
The whitespace used here is a bit of a mess.
Maybe ```_drawCount++``` instead? (There shouldn't be any difference between the two and it is nicer to read).

Sorry just nitpicking really :)